docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,116 @@
+# .NET Standard vNext
+
+This document describes the plan for .NET Standard 2.1, which includes the
+definition of its API surface.
+
+## Goals
+
+* **Include `Span<T>`**. In .NET Core 2.1 we've added `Span<T>` which is an
+  array like type that allows representing managed and unmanaged memory in a
+  uniform way and supports cheap slicing. It's at the heart of most performance-
+  related improvements in .NET Core 2.1 and allows managing buffers in a more
+  efficient way as it helps in reducing allocations and copying. `Span<T>` is
+  considered a primitive type and requires runtime and compiler support in order
```

Comment:
Is `Type.IsPrimitive` going to return `true` for `Span<T>`? If so, this is okay - otherwise I'd choose a different word to avoid confusion.